esx_service/vmdk_ops.py

```diff
@@ -808,7 +821,9 @@ def handleVmciRequests(port):
             logging.info("executeRequest '%s' completed with ret=%s",
                          req["cmd"], ret)
 
-        response = lib.vmci_reply(c, c_char_p(json.dumps(ret)))
+            ret_string = json.dumps(ret)
```

Comment:
Should these 2 lines really be moved inside the else clause? If so, response will be None if there is an exception and we will never match `if response == VMCI_ERROR:`